Ibistic.Public.OpenAirportData/Ibistic.Public.OpenAirportData/Airport.cs

```diff
@@ -0,0 +1,24 @@
+﻿namespace Ibistic.Public.OpenAirportData
+{
+    public sealed class Airport
+    {
+        public string Name { get; internal set; }
+        public string City { get; internal set; }
+        public string CountryName { get; internal set; }
```

Comment:
It seems as if Country and Time Zone might be optional, so in that case it could perhaps be useful to have a couple of properties a long the lines of HasTimeZone - HasCountry or similar. Mainly as these concepts are made up of multiple variables.

It also not obvious to me why you have a Country structure, but you are not using it here.

Reply:
Really, all concepts are optional and null is used to represent "none". This library leaves validation and filtering of "usable" airports up to the client. A provider _could_ provide any random combination of the three TimeZone properties. for example.